Gulpfile.js

```diff
@@ -1,33 +1,42 @@
-var autoprefix = require("gulp-autoprefixer"),
```

Comment:
Upon contemplating this, I support this change as it is more true to the actual output of the library.